src/pages/events.js

```diff
@@ -44,205 +56,360 @@ export const StyledPromoteSection = styled.div`
   }
 `;
 
+export const StyledEvents = styled.ul`
+  display: flex;
+  flex-direction: column;
+  gap: 48px;
+`;
+
+export const StyledFormContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 60px;
+  align-items: flex-start;
+`;
+
+export const StyledFormGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 60px 64px;
+  width: 100%;
+
+  ${mQ(bp.tablet)} {
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  > *:last-child {
+    grid-column: 1/-1;
+  }
+`;
+
 const Events = () => {
+  const theme = useTheme();
+
+  const [formData, setFormData] = useState({
+    name: '',
+    email: '',
+    country: '',
+    city: '',
+    eventName: '',
+    eventType: '',
+    eventDate: '',
+    eventTime: '',
+    eventLocation: '',
+  });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormData((prevData) => ({
+      ...prevData,
+      [name]: value,
+    }));
+  };
+
+  const [success, setSuccess] = useState(false);
+  const [error, setError] = useState(null);
+  const [disabled, setDisabled] = useState(false);
+
+  // Handle form submission
+  const form = useRef();
+  const submit = useCallback(async (e) => {
+    e.preventDefault();
+    if (disabled) return;
+    setDisabled(true);
+    setError(null);
+    setSuccess(false);
+
+    if (!form.current?.reportValidity()) {
+      setDisabled(false);
+      setError('Please fill out all the fields.');
+      return;
+    }
+
+    console.log(formData);
```

Comment:
Form appears to be lacking mechanics for actually submitting to anywhere

Reply:
Form removed per request